asserts/ifacedecls.go

```diff
@@ -150,6 +150,14 @@ func (matcher mapAttrMatcher) feature(flabel string) bool {
 
 func (matcher mapAttrMatcher) match(apath string, v interface{}, ctx AttrMatchContext) error {
 	switch x := v.(type) {
+	case Attrer:
```

Comment:
thanks, after I remembered that lists in the pattern represent alternatives, this is the right thing to do